Add support for new fields in external services template (#2348)
strimzi/strimzi-kafka-operator: documentation/modules/con-customizing-external-services.adoc

// This assembly is included in the following assemblies:
//
// assembly-customizing-deployments.adoc

[id='con-customizing-external-services-{context}']
= Customizing external Services

> When exposing Kafka to the outside of Kubernetes using load balancers or node ports, you can customize some more fields in addition to Labels and Annotations.

Reviewer: ```suggestion
When exposing Kafka outside of Kubernetes using loadbalancers or node ports, you can use additional customization properties in addition to labels and annotations.
```

> When exposing Kafka to the outside of Kubernetes using load balancers or node prots, you can in addition to Labels and Annotations customize some more fields .

Reviewer 2: typo "prots".
"you can customize some more fields in addition to Labels and Annotations" sounds better to me, but it's more for @PaulRMellor or @laidan6000

> These fields are described in the following table and affect how the Service is created.

Reviewer: ```suggestion
The properties for external services are described in the following table and affect how the service is created.
```

[table,stripes=none]
|===
|Field |Description

|`externalTrafficPolicy`

> |Denotes if this Service desires to route external traffic to node-local or cluster-wide endpoints.

Reviewer: ```suggestion
|Specifies whether the service routes external traffic to node-local or cluster-wide endpoints.
```

`Cluster` may cause a second hop to another node and obscures the client source IP.
`Local` avoids a second hop for LoadBalancer and Nodeport type services and preserves the client source IP (when supported by the infrastructure).

> When not specified, Kubernetes will use `Cluster` as default.

Reviewer: ```suggestion
If unspecified, Kubernetes will use `Cluster` as the default.
```

|`loadBalancerSourceRanges`

> |A list of CIDR ranges (for example `10.0.0.0/8` or `130.211.204.1/32`) from which clients will be able to connect to load balancer type listeners.

Reviewer: ```suggestion
|A list of CIDR ranges (for example `10.0.0.0/8` or `130.211.204.1/32`) from which clients can connect to load balancer type listeners.
```

> If supported by the platform, this will restrict traffic through the load balancer to the specified CIDR ranges in this list.

Reviewer: ```suggestion
If supported by the platform, traffic through the loadbalancer is restricted to the specified CIDR ranges.
```

> This field is applicable only for load balancer type services and will be ignored if the cloud-provider does not support the feature.

Reviewer: ```suggestion
This field is applicable only for loadbalancer type services, and is ignored if the cloud provider does not support the feature.
```

> For more info see link:https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/[Configure Your Cloud Provider's Firewalls].

Reviewer: ```suggestion
For more information, see https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/.
```

|===

> These fields are available on the `externalBootstrapService` and `perPodService` fields.

Reviewer: ```suggestion
These properties are available for `externalBootstrapService` and `perPodService`.
```

> The following example shows these customized fields on a `template` property:

Reviewer: ```suggestion
The following example shows these customized properties for a `template`:
```
